qcfractal/alembic/versions/1bd2f7711e68_wavefunction_table.py

```diff
@@ -0,0 +1,53 @@
+"""Wavefunction data table
+
+Revision ID: 1bd2f7711e68
+Revises: a2f76bb7be65
+Create Date: 2019-10-17 14:03:07.364322
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import qcfractal
+
+
+# revision identifiers, used by Alembic.
+revision = '1bd2f7711e68'
+down_revision = 'a2f76bb7be65'
+branch_labels = None
+depends_on = None
+
+
```

Comment:
Doesn't need to be in this PR, but we should get localized orbitals in. They're useful for education!